.github/workflows/win32.yml

```diff
@@ -68,14 +68,16 @@ jobs:
         cp doc/070_packaging.md publish/quake2-win32-${{github.sha}}/misc/docs/070_packaging.txt
         cp doc/080_contributing.md publish/quake2-win32-${{github.sha}}/misc/docs/080_contributing.txt
         cp doc/090_filelists.md publish/quake2-win32-${{github.sha}}/misc/docs/090_filelists.md
-        # SDL2
-        wget -c https://github.com/libsdl-org/SDL/releases/download/release-2.32.8/SDL2-2.32.8-win32-x86.zip
-        unzip -o SDL2-2.32.8-win32-x86.zip
-        cp SDL2.dll publish/quake2-win32-${{github.sha}}/
+        # SDL3
+        gh release download release-3.2.22 \
```

Comment:
Unfortunately GitHub has not provided 32 bit version of their tool, so only wget/curl solution possible.

Reply:
Great.